cds/config.py

```diff
@@ -1005,3 +1007,16 @@ def _(x):
     'streams/0/creation_time': ['format/tags/creation_time'],
 }
 CDS_FFMPEG_METADATA_POST_SPLIT = ['streams/0/keywords']
+
+###############################################################################
+# LOG USER ACTIVITY
+###############################################################################
+
+LOG_ACTIONS_BASE_URL = 'http://cds-test-wn-02.cern.ch/tools/events.py/'
```

Comment:
``//`` instead of ``http://``? It might causes issues with the secure content policy, WDYT?

Reply:
fixed